salt/states/win_wua.py

```diff
@@ -428,7 +428,8 @@ def uptodate(name,
     ret = {'name': name,
            'changes': {},
            'result': True,
-           'comment': ''}
+           'comment': '',
+           'not_installed': ''}
```

Comment:
This return is a standardized return used by the state system, so we shouldn't add new fields here. Rather than adding `not_installed` here, this should just be appended to the text in the `comment` field.